metadata/AustralianInvoices__8__metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "AustralianInvoices",
+    "retrainable": true,
+    "gpu": 1,
+    "processorType": "GPU",
+    "description": "Machine Learning model(available in Preview) for extracting commonly occurring data points from Invoices from Australia, including header fields and line items. Please see more details including supported languages in the ML Packages section of the Document Understanding Guide: https://docs.uipath.com/document-understanding/docs/ml-packages",
+    "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -\u003e Other Services view.",
+    "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+    "changeLog": "Model: 21.10.10",
+    "cpu": 0,
+    "inputType": "JSON",
+    "displayName": "AustralianInvoices",
+    "memory": 0,
+    "mlPackageLanguage": "PYTHON37_DU",
+    "projectId": "[project-id]",
+    "stagingUri": "[staging-uri]",
+    "projectName": "UiPath Document Understanding",
+    "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+    "tenantName": "UiPath",
+    "minAIFabricVersion": "v21.10",
```

Comment:
@vivekjhauipath same min version?

Reply:
yes, because these will be part of new baseImage. Though we can potentially use 21.4 too.

Reply:
but testing is only valid for 21.10 AIC